t/twig/twig_1.42.4_1.42.5_rhel_8.4.sh

```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
+# -----------------------------------------------------------------------------
+#
+# Package       : twig/twig
+# Version       : 1.42.4, 1.42.5 
```

Comment:
As per line 18, build script is created for version v1.42.4.  Please update accordingly

Reply:
The script is tested and working fine on rhel 8.4. Also parameterized the version in line 18.